docker-compose.yml

```diff
@@ -1,13 +1,26 @@
 version: '3'
 
 services:
-    browser_fpm:
-        image: ${DOCKER_NAMESPACE}/browser:${REVISION_BROWSER}
     dummy-api_fpm:
         image: ${DOCKER_NAMESPACE}/dummy-api:${REVISION_DUMMY_API}
+        volumes:
+            - public-api:/app/public
+    browser_fpm:
+        image: ${DOCKER_NAMESPACE}/browser:${REVISION_BROWSER}
+        volumes:
+            - public-browser:/app/public
     web:
         image: nginx:1.15.2-alpine
         volumes:
             - .docker/public-nginx.conf:/etc/nginx/conf.d/default.conf
+            - public-api:/app/api/public
```

Comment:
Try to restore the `try_files` for the API?

Reply:
But doesn't serve files at the moment? Could do though

Reply:
Volume's also not needed, but more consistent to serve (in case we forget).